### DISCUSSION TITLE
How do i easily add view and worldmodels?

### DISCUSSION THREAD

**Discussion by Blechkanne** [Alf21/TTT2#1430] [Q&A]
i tried out the new feature with SWEP:AddCustomViewModel & AddCustomWorldModel.
First of all Is there a list of existing view or worldmodels?
I am currently doing manipulation of the viewmodel by changing values in this settings object
```
 {
            type = "Model",
            model = "... .mdl",
            bone = "ValveBiped.Bip01_R_Finger2",
            rel = "",
            pos = Vector(3, 5, 3),
            angle = Angle(180, 80, 20),
            size = Vector(0.3, 0.3, 0.3),
            color = Color(255, 255, 255, 255),
            surpresslightning = false,
            material = "",
            skin = 0,
            bodygroup = {},
}
```
Then i am reloading the map, making myself traitor, buying myself the weapon to see the impact of the changes.
Please tell me there is an easier way. I found some Bone Tools but i have no idea how they would work with c_model hands

**Comment by saibotk** [comment]
@TimGoll can you help out here?
